src/pcap.rs

```diff
@@ -1,41 +1,18 @@
 //! Reading packets as events from a pcap input.
 
-use crate::Error;
+use crate::{BareEvent, Error};
 use std::io::{self, Read};
 
 use pcap_parser::{
     create_reader, data::get_packetdata_ethernet, data::PacketData, traits::PcapReaderIterator,
     Block, PcapBlockOwned, PcapError,
 };
 
-/// A packet in a pcap input.
-#[derive(Debug)]
-pub struct Event {
```

Comment:
How about adding a type alias, e.g., `type Event = BaseEvent`? Then a crate user can write `pcap::Event` or `text::Event`, and doesn't have to know which input modules use `BareEvent` and which ones don't.